tests/project-type/workspace-instance.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { WorkspaceImpl } from "@sap/project-api";
+import { expect } from "chai";
+import { uniq } from "lodash";
+
+import { mockVscode } from "../mockUtil";
+
+const testVscode = {
+    workspace: {
+        workspaceFolders: () => []
+    }
+};
+
+mockVscode(testVscode, "src/project-type/workspace-instance.ts");
+import { getBasWorkspaceAPI } from "../../src/project-type/workspace-instance";
+
+describe.skip("getBasWorkspaceAPI", () => {
+    it("multiple bas workspace instances should be equal", () => {
+        const basWsInstances = [];
+        for (let i = 0; i < 5; i++) {
+            basWsInstances.push(getBasWorkspaceAPI(WorkspaceImpl));
+        }
+        expect(uniq(basWsInstances)).to.have.lengthOf(1);
```

Comment:
It is really nice you are testing for multiple (5) invocation results.
However this does make the assertion less understandable (how vs what).
It should be possible to write something like:
```javascript
const firstBasWsApi = getBasWorkspaceAPI();
expect(basWsInstances).to.each.equal(firstBasWsApi);
```

but that would require adding another chai plugin which is likely overkill if we only do this assertion once:
- https://www.chaijs.com/plugins/chai-each/

Reply:
I will add the plugin once we have one more similar test